src/novatel_oem7_driver/config/std_msg_topics.yaml

```diff
@@ -1,5 +1,6 @@
 # ROS-standard
 IMU:        {topic: /gps/imu,                 frame_id: imu_link}
+RAWIMU:     {topic: /imu/data_raw,            frame_id: imu_link}
```

Comment:
ROS convention is the `/imu/raw` topic.

Reply:
Thank you for your input. We follow the convention based on the following reference: https://www.ros.org/reps/rep-0145.html

imu/data_raw is suggested in the above reference. Would you be able to provide clarification on the convention of /imu/raw?

Reply:
Ow damn, I was mistaken here! My bad